rich-text.gemspec

```diff
@@ -22,7 +22,8 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "diff-lcs", "~> 1.2.5"
   spec.add_dependency "activesupport", ">= 3.0.0"
-  # spec.add_dependency "nokogiri", ">= 1.0.0"
+  spec.add_dependency "nokogiri", ">= 1.10.8"
```

Comment:
It will make people's lives easier if we were as relaxed about this dependency as possible. Are you using any specific feature of nokogiri that's not available in previous 1.x versions?